docs/modules/ROOT/pages/vshn-managed/mariadb/restore.adoc

```diff
@@ -0,0 +1,4 @@
+= Restores
+
+Self-service restores aren't implemented yet. If you need to restore a database
+please contact our support: support@vshn.ch.
```

Comment:
maybe remove the dot after `ch.` - it might be taken as part of the email if someone clicks address

Reply:
I've replaced it with a proper mailto link, can you have another look?